src/open_inwoner/openklant/api_models.py

```diff
@@ -6,13 +6,19 @@
 from zgw_consumers.api_models.base import ZGWModel
 
 
-class KlantCreateData(TypedDict):
-    bronorganisatie: str
-    voornaam: str
-    voorvoegselAchternaam: str
-    achternaam: str
+class KlantWritePayload(TypedDict, total=False):
```

Comment:
Perhaps the name itself (not just the comment) should make clear that this is intended for eSuite: `KlantWritePayloadESuite` or the like?